### DISCUSSION TITLE
Tuto for Synology for beginner

### DISCUSSION THREAD

**Discussion by hercut** [karakeep-app/karakeep#698] [Ideas]
Hello,

First of all, thank you and congratulations on your work—your application is truly outstanding! I already use Wallabag, Shaarli, and Joplin, and your solution seems to be the most complete for my needs. However, as a beginner, I would greatly appreciate some assistance.

I recently installed your application on my Synology using information I found online. The installation appears to have worked, and everything is functioning well so far. However, I’m not entirely sure if I installed it correctly.

I’d also like to update the application to the latest version, but I haven’t been able to find clear instructions on how to do this. Could you help me with this and, perhaps, assist other users who might be in the same situation?

Finally, if I may make a small suggestion: it would be great to have a feature that allows hiding or creating a hidden folder, which could be useful for users like me. 😊

Thank you again for your help and for the amazing work you’ve done.

Best regards,

---

Bonjour,

Tout d’abord, merci et bravo pour votre travail : votre application est un outil remarquable ! J’utilise déjà Wallabag, Shaarli et Joplin, et votre solution me semble la plus complète pour mes besoins. Cependant, je débute dans ce domaine, et j’aurais besoin d’un peu d’aide.

J’ai récemment installé votre application sur mon Synology en m’appuyant sur des informations trouvées sur Internet. L’installation semble avoir réussi, et tout fonctionne bien pour l’instant. Cependant, j’ai quelques doutes sur la conformité de cette installation.

Je souhaiterais également mettre à jour l’application vers la dernière version, mais je ne parviens pas à trouver de guide ou d’informations claires pour effectuer cette opération. Pourriez-vous m’aider, et peut-être par la même occasion aider d’autres utilisateurs dans la même situation ?

Enfin, si je peux me permettre une petite suggestion : il serait intéressant d’ajouter une fonctionnalité pour cacher ou créer un répertoire masqué, ce qui pourrait être utile pour certains utilisateurs comme moi. 😊

Encore merci pour votre aide et votre excellent travail.

**Comment by kriskras69** [comment]
I'm not sure if it's allowed to post links to other sites but if not i'm sure they can remove it.
No expert myself at all but i think this is a good place for you to find info, also about hoarder on Nas.
https://mariushosting.com/
Hope it helps, have a good one.
Respect!
